Scripts/DCS-BIOS/lib/modules/aircraft_modules/MiG-29A.lua

```diff
@@ -238,6 +238,9 @@ MiG_29A:defineFloat("AOA_G_METER_AOA_POINTER", 7, { 0, 1 }, AOA_G_METER, "Curren
 -- ADF mode toggle switch
 
 -- Nose wheel brake
+local NOSE_WHEEL_BRAKE = "Nose Wheel Brake"
```

Comment:
I'm not sure what to call this, but I think calling it "Nose Wheel Brake" could maybe be confusing. I know that's what it's called the game _and_ in the manual, but... I could see people looking here for the brake _lever_, even though that control would be under the Stick category. This handle actually just enables or disables the nose wheel braking system (genuinely curious why it's so prominent in the center of the instrument panel... were there constantly issues with the system?). It doesn't actually activate the brakes, which is what I'd initially assume.

Maybe "Nose Wheel Brake Handle (Instrument Panel)" or something?

<img width="762" height="1041" alt="Image" src="https://github.com/user-attachments/assets/a4303313-2ee1-4377-83c9-ab89ae832458" />

Reply:
I was more suggesting updating the category name rather than the control description actually